core/equations.gms

```diff
@@ -55,7 +55,7 @@ q_costInv(t,regi)..
 q_costInvTeDir(t,regi,te)..
   v_costInvTeDir(t,regi,te)
   =e=
-  vm_costTeCapital(t,regi,te) * sum(te2rlf(te,rlf), vm_deltaCap(t,regi,te,rlf) )
+  vm_costTeCapital(t,regi,te) * sum(te2rlf(te,rlf), vm_deltaCap(t,regi,te,rlf)) * 1.05**(pm_ts(t) / 2)
```

Comment:
could you add a line to the equation comments explaining the 1.05 value here? Could this number be included as a scalar/parameter instead of hard codded to the equation?

Reply:
I changed it to: 

  * (1.02 + pm_prtp(regi) ** (pm_ts(t) / 2) !! This increases the investments as if the money was actually borrowed 
  !! half a time step earlier, using an interest rate of pm_prtp + 2%, which is close to the model-endogenous interest rate. 
  !! We do this to reduce the difference to the previous version where the effect of deltacap on capacity was split 
  !! half to the current and half to the next time. 

is this ok?